kubernetes/k8s-deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: atomicred
+  namespace: atomic-red
+  labels:
+    app: atomicred
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: atomicred
+  template:
+    metadata:
+      labels:
+        app: atomicred
+    spec:
+      containers:
+      - name: atomicred
+        image: issif/atomic-red:latest
```

Comment:
Hello @nigel-falco . Thanks for your contribution. We already have a container image(`redcanary/invoke-atomicredteam`) in docker hub. You can find more info about that [here](https://github.com/redcanaryco/invoke-atomicredteam/wiki/Docker-Containers)

Can you rename `issif/atomic-red:latest` to `redcanary/invoke-atomicredteam` ?

Reply:
Yes! @cyberbuff I submitted an updated PR with the official container image.

Thank you!